CMakeLists.txt

```diff
@@ -105,6 +106,7 @@ else()
     # Note this definition doesn't appear in the cache variables.
     add_definitions(-DNDEBUG)
 endif()
+add_feature_info(Assertions ENABLE_ASSERTIONS "Enables assertions")
```

Comment:
I just tried this; I think these messages need to *not* start with an upper-case, otherwise it reads weirdly with a capital letter in the middle.

Reply:
I also thought about it. But in the [official example](https://cmake.org/cmake/help/latest/module/FeatureSummary.html) they use upper-case, do not know why. But we can use lower-case. What do you think?

Reply:
Well, now you've linked me to that, maybe we should just leave it. If you care, we can. If you don't, I'm not _that_ worried.

Reply:
Just checked [other projects](https://github.com/search?p=3&q=add_feature_info&type=Code), sometimes people start the description with lower-case, sometimes with upper-case. So I do not know how to make it right. I personally use upper-case as in the official examples. But it's up to you to decide.

Reply:
Leave it; it really is unclear which way is "best". I might tidy them up later on to work better with leading lower-case -- right now, they're just sentence fragments so it doesn't matter.